students/terrance_jones/lesson2/fizzbuzzUPDATED.py

```diff
@@ -1,22 +1,17 @@
-# To change this license header, choose License Headers in Project Properties.
-# To change this template file, choose Tools | Templates
-# and open the template in the editor.
 def fizzbuzz():
-	''' Fizzbuzz prints fizz for numbers
-	divisible by 3, prints buzz for numbers divisible by 5
-	and prints fizzbuzz for numbers divisible by 3 and 5. All others 
-	it prints the number'''
-        
-    for i in range(100):
-	if i == 0:
-            print()
-		elif(i%3 == 0 and i%5!=0):
-            print("Fizz")
-	elif(i%5== 0 and i%3!=0):
-		print("Buzz")
-	elif(i%3==0 and i%5==0):
-		print("FizzBuzz")
-	else:
-            print(i)
+    '''Fizzbuzz prints fizz for numbers divisible by 3, prints buzz for numbers
+    divisible by 5 and prints fizzbuzz for numbers divisible by 3 and 5. All 
+    other it prints the number'''
 
-        i = i + 1
+        for i in range(100):
```

Comment:
the indentation is now 4 spaces. but there should be no indentation after your docstrings, like:
```
def some_method():
    """this is a function
    second line"""
    for i in range():
        pass
```
also notice that docstrings use triple double quotes.